### DISCUSSION TITLE
Getting CSLA0010 Arguments should be Serializable for Nullable Type

### DISCUSSION THREAD

**Discussion by mtavares628** [MarimerLLC/csla#2617] [Questions]
I'm passing Nullable<int> parameters into my DataPortal Fetch method, and the analyzer warning that operational arguments must be serializable.  Based on looking at the definition of Nullable<T> it is marked with the Serializable attribute.  Is this an issue with the analyzer, or are Nullable Types not serializable with the DataPortal methods?

**Comment by rockfordlhotka** [comment]
You know @mtavares628, I don't know that I've ever tested to see if MobileFormatter properly serializes Nullable types. I _think_ so, but I'm not 100% sure.

Right this moment, @TheCakeMonster has been working to get the unit tests running again thanks to all the changes in CSLA 6...

I do suspect that Nullable types serialize fine, and it is the analyzer that needs to be updated.

I've created a backlog item: https://github.com/MarimerLLC/csla/issues/2619
